<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentProgressCheck</class>
 <widget class="QDialog" name="studentProgressCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:light gray</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>421</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Student Degree Progression</string>
   </property>
  </widget>
  <widget class="QLabel" name="completedCoursesTitle">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Completed Courses</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="completedCourseTable">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>481</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Academy Engraved LET</family>
     <bold>false</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>570</y>
     <width>461</width>
     <height>101</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>49</number>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalCredits">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Credits</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculateProgress">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>480</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Calculate Progress</string>
   </property>
  </widget>
  <widget class="QLabel" name="degreeEntryTitle">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Degree/Faculty</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="degreeEntry">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalCreditsEntry">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="resultOutcome">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>520</y>
     <width>441</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="errorMessageBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="id">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
